### PR TITLE
Answers-fix-required

### PR DESCRIPTION
**Pull request by underchemist**
# Description

Please include a summary of the change. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Instructions

Explain what someone needs to do in order to test the functionality of the changes. 

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by underchemist** [comment]
- adds or removes instances of `<RequiredIndicator />` in react form (shows red asterix)
- update `ValidationSchema` for questions that need to handle `null` values for now not required questions

File: mrtt-ui/src/components/ManagementStatusAndEffectiveness/ManagementStatusAndEffectivenessForm.js [hunk 1]
```diff
@@ -170,6 +171,7 @@ const ManagementStatusAndEffectivenessForm = () => {
       <Form>
         <FormQuestionDiv>
           <StickyFormLabel>{questions.dateOfAssessment.question}</StickyFormLabel>
+          <RequiredIndicator />
```
Comment (ukhan-sg) [inline mrtt-ui/src/components/ManagementStatusAndEffectiveness/ManagementStatusAndEffectivenessForm.js:174]: image

Can you nest `<RequiredIndicator />` inside StickyFormLabel so it looks like:
```
<StickyFormLabel>
  {questions.dateOfAssessment.question}
  <RequiredIndicator />
</StickyFormLabel>
```

Should make the formatting match other questions.
image

File: mrtt-ui/src/components/SocioeconomicAndGovernance/SocioeconomicAndGovernanceStatusAndOutcomesForm.js [hunk 2]
```diff
@@ -277,6 +278,7 @@ const SocioeconomicAndGovernanceStatusAndOutcomesForm = () => {
       <Form>
         <FormQuestionDiv>
           <StickyFormLabel>{questions.dateOfOutcomesAssessment.question}</StickyFormLabel>
+          <RequiredIndicator />
```
Comment (ukhan-sg) [inline mrtt-ui/src/components/SocioeconomicAndGovernance/SocioeconomicAndGovernanceStatusAndOutcomesForm.js:281]: Same comment about nesting inside `StickyFormLabel`

File: mrtt-ui/src/components/EcologicalStatusAndOutcomes/EcologicalStatusAndOutcomesForm.js [hunk 3]
```diff
@@ -396,6 +397,7 @@ const EcologicalStatusAndOutcomesForm = () => {
       <Form>
         <FormQuestionDiv>
           <StickyFormLabel>{questions.dateOfEcologicalMonitoring.question}</StickyFormLabel>
+          <RequiredIndicator />
```
Comment (ukhan-sg) [inline mrtt-ui/src/components/EcologicalStatusAndOutcomes/EcologicalStatusAndOutcomesForm.js:400]: Same comment about nesting inside StickyFormLabel

**Review by ukhan-sg — APPROVED** [review]
Looks great! 👍
Thanks for the changes.

Left minor comments to fix the required indicator formatting.

File: mrtt-ui/src/components/SiteInterventions/SiteInterventionsForm.js [hunk 4]
```diff
@@ -377,6 +381,7 @@ function SiteInterventionsForm() {
           <ErrorText>{errors.whichStakeholdersInvolved?.selectedValues?.message}</ErrorText>
         </FormQuestionDiv>
         <FormQuestionDiv>
+          <RequiredIndicator />
           <CheckboxGroupWithLabelAndController
```
[the remark this answers is not in the document]
Comment (underchemist) [reply mrtt-ui/src/components/SiteInterventions/SiteInterventionsForm.js:385]: My concern was that component is reused elsewhere, so I don't want to mark all questions using this component as required. Is there a way to set that conditionally?

**Comment by underchemist** [comment]
closing in favour of #376

**Pull request closed without merging**